modules/ROOT/pages/clustering/setup/discovery.adoc

```diff
@@ -84,4 +188,264 @@ For further information, see the Kubernetes documentation on https://kubernetes.
++
+[queryresult]
+----
++------------------------------------------------------------------------+
+| mode         | stateComparison                                         |
++------------------------------------------------------------------------+
+| "V1_OVER_V2" | "States are not matching after PT1M24.778S: (score:18)" |
++------------------------------------------------------------------------+
+----
++
+The score is a measure of how different the states are.
+The score is 0 when the states are matching.
+When some members are running just one of the discovery services (v1 or v2) and other members run both, the score stays permanently high.
+This is no reason for worry.
+
+
+. To fulfill this convergence, in different terminals, connect to server02 and server03 via `bolt://` and repeat steps 3 and 4 on both of them.
+
+. Check the state on all servers again.
+It should show that the states are `Matching`.
++
+[queryresult]
+----
++--------------------------------+
+| mode         | stateComparison |
++--------------------------------+
+| "V1_OVER_V2" | "Matching"      |
++--------------------------------+
+----
+
+. On all three servers, run:
++
+[source,cypher]
+----
+CALL dbms.cluster.switchDiscoveryServiceVersion("V2_OVER_V1");
+----
++
+At this point, v2 is the service that is running the cluster, with v1 running in the background.
++
+[queryresult]
+----
++--------------------------------+
+| mode         | stateComparison |
++--------------------------------+
+| "V2_OVER_V1" | "Matching"      |
++--------------------------------+
+----
+
+
+. Finally, turn off v1 by running the following procedure on all three servers:
++
+[source,cypher]
+----
+CALL dbms.cluster.switchDiscoveryServiceVersion("V2_ONLY");
+----
+
+.  Verify that `CALL dbms.cluster.showParallelDiscoveryState();` now shows `V2_ONLY` running.
+Note that `stateComparison` is `N/A` because you do not have v1 to compare states anymore.
++
+[queryresult]
+----
++-----------------------------+
+| mode      | stateComparison |
++-----------------------------+
+| "V2_ONLY" | "N/A"           |
++-----------------------------+
+----
++
+.Important
+[IMPORTANT]
+====
+Remember to update the _neo4j.conf_ files for all the servers.
+The switching using procedures does not persist anything to disk.
+Therefore, when a server restarts, it starts right back with only v1 running.
+As such, ensure that `dbms.cluster.discovery.version=V2_ONLY`, and that `dbms.cluster.discovery.v2.endpoints` or `dbms.kubernetes.discovery.v2.service_port_name`
+are set as required, so that the servers start with v2 running on the next restart.
+====
+
+
+
+== Monitoring the move
+
+When moving from the current discovery service v1 to the new version v2, you can monitor the progress using the following metrics:
+
+* xref:monitoring/metrics/reference.adoc#discovery-service-V1[Discovery metrics v1]
+* xref:monitoring/metrics/reference.adoc#metrics-discovery-v2[Discovery metrics v2]
```

Comment:
Sorry, one more comment, I just noticed that we are not exposing `difference_score` in the metrics, only in the procedure. So I'm inclined for this part to be removed.